Cargo.lock

```diff
@@ -419,18 +419,17 @@ checksum = "baf1de4339761588bc0619e3cbc0120ee582ebb74b53b4efbf79117bd2da40fd"
 
 [[package]]
```

Comment:
I've updated our version of chrono because of this error when compiling plotters: https://github.com/plotters-rs/plotters/issues/624

An example of why using a lock file is very helpful 👍